Generate the `activation` ping and send it with Glean (#1707)
mozilla-mobile/fenix: docs/activation.md

# The `activation` ping

## Description

> This ping is intended to provide a measure of the activation of mobile products.

Reviewer: ```suggestion
This ping provides a measure of the activation of mobile products.
```

Reviewer: "intended to" sounds so passive. It is sent and it is as reliable as other pings.

## Scheduling

> The `activation` ping is automatically sent at startup, after Glean is initialized, and contains

Reviewer: ```suggestion
The `activation` ping is automatically sent at first startup, after Glean is initialized, and contains
```

Reviewer: We should make this clear this ping is sent only once, only during the very first startup (or during subsequent startups if it hasn't been sent yet?).

> the following fields:

Reviewer: Don't mention this in the scheduling section? It is its own section anyway.

## Contents
| Field name | Type | Description |
|---|---|---|
| `identifier` | String | An hashed and salted version of the Google Advertising ID from the device. |
| `activation_id` | UUID | An alternate identifier, not correlated with the client_id, generated once and only sent with the activation ping. |

> The `activation` ping also includes the common [ping sections]https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/pings.md#ping-sections)

Reviewer: ```suggestion
The `activation` ping also includes the common [ping sections](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/pings.md#ping-sections)
```
